server/src/visitors/YmlFunctionVisitor.ts

```diff
@@ -49,10 +126,7 @@ export class YmlFunctionVisitor extends YmlBaseVisitor {
     const functionNameSubParts = fullName.split("::");
     if (functionNameSubParts.length > 1) {
       methodName = functionNameSubParts.pop();
-      className = fullName.replace(
-        new RegExp(`::${methodName}$`, "u"),
-        "",
-      );
+      className = fullName.replace(new RegExp(`::${methodName}$`, "u"), "");
```

Comment:
* Old code, but still, aren't we compiling a regexp several times for nought, here?
* On my Prism.js PR, I was told (in js) that `RegExp(…)` ⇔ `new RegExp(…)` and that the shorter form was preferred. See if it works the same way in ts.

Reply:
I prefer the version with because of `Java` (bad?) habits. It makes clearer to me that we are creating a new object.
The regexp is dynamic. We then compile it only once for each function/method.